src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -129,26 +128,26 @@ function adjustStudentsPanelView($panelHeading, panelCollapse,
 function expandCollapseExistingStudentsPanel() {
     const $panelHeading = $(this);
     const panelCollapse = $panelHeading.parent().children('.panel-collapse');
-    const displayIcon = $panelHeading.children('.display-icon');
+    const displayStatus = $panelHeading.children('.display-status');
     const toggleChevron = $panelHeading.parent().find('.glyphicon-chevron-down, .glyphicon-chevron-up');
```

Comment:
Now that I've looked at this more closely, I think you only need to pass `$panelHeading` (btw not sure why you add `$` at the front of this variable name) and `displayStatus` here. The other variables can be resolved when they are needed down the call chain.

Reply:
Noted 😄 

Actually a lot of code regarding panel behavior was inspired from the current `instructorStudentList.js`, more specifically the `seeMoreRequest` function. Should probably note that existing JavaScript functions might need some refactoring work if not other developers will propagate similar changes.